app/services/widgets/settings/alert-box/index.ts

```diff
@@ -211,6 +212,10 @@ export class AlertBoxService extends WidgetSettingsService<IAlertBoxData> {
       setting.image_href === '/images/gallery/default.gif'
         ? 'http://uploads.twitchalerts.com/image-defaults/1n9bK4w.gif'
         : setting.image_href;
+    const constrainedDuration =
+      Math.floor(setting.alert_duration / 1000) <= 300
+        ? Math.floor(setting.alert_duration / 1000)
+        : 300;
```

Comment:
const constrainedDuration = Math.floor(Math.max(300, setting.alert_duration / 1000))

Reply:
i dont want to hold up preview so i'll make a note of that and make this change in a future PR